apps/docs/src/guide/transactions/pre-confirmation.md

```diff
@@ -0,0 +1,60 @@
+# Pre Confirmation
+
+## What is Pre Confirmation?
+
+**Pre confirmation** is an intermediate transaction status that occurs after a transaction has been **submitted** and **accepted** by the blockchain, but **before** it is fully **processed and included** in a new block.
+
+At this stage, the transaction is pre-executed and assigned one of two possible outcomes:
+
+- **PreconfirmationSuccessStatus**:
```

Comment:
According to regular capitalization and _camelization_ patterns, the `c` in `Confirmation` should be uppercase.

I'm commenting here, but this would be a big search'n'replace in case you agree.

```suggestion
- **PreConfirmationSuccessStatus**:
```

Reply:
@arboleya I used this approach because the status name on the fuel-core side is [PreconfirmationSuccessStatus](https://github.com/FuelLabs/fuels-ts/blob/6f9e3abe2f772a98f883458830eb72f59412d7b1/packages/account/src/providers/transaction-summary/types.ts#L69)

Reply:
Hmm got it. This can be easily misunderstood, but there's no quick solution, so it's ok.